components/dashboard/src/menu/Menu.tsx

```diff
@@ -178,23 +192,17 @@ const UserMenu: FC<UserMenuProps> = ({ user, className, withAdminLink, withFeedb
                 link: "/admin",
             });
         }
-        if (withFeedbackLink && isGitpodIo) {
-            items.push({
-                title: "Feedback",
-                onClick: onFeedback,
-            });
-        }
 
         // Add a separator to the last item
         if (items.length > 0) {
             items[items.length - 1].separator = true;
         }
 
         return items;
-    }, [isGitpodIo, onFeedback, user?.rolesOrPermissions, withAdminLink, withFeedbackLink]);
+    }, [user?.rolesOrPermissions, withAdminLink]);
 
     const menuEntries = useMemo(() => {
-        return [
+        const baseEntries: ContextMenuEntry[] = [
```

Comment:
nit: let's just call these `entries`, since we're pushing to them instead of creating some new variable with them

Reply:
Resolved in 657d73f